db/migrate/20230401123456_create_styles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStyles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :styles do |t|
+      t.string :name, null: false
+      t.references :library, null: false, foreign_key: true
```

Comment:
The 'styles' and 'components' tables are being created with 'library_id' as a foreign key, but there are no indices on 'library_id'. Consider adding an index to improve query performance.